### ISSUE TITLE
Feature: Heatmap

### ISSUE THREAD

**Issue by ajkeith** [treypujats/COVID19#33]
I believe this is about converting the summary tab into a visual? @tleonard6 has more details.

**Comment by tleonard6** [comment]
Yes, that's exactly what we're going for here.  Goal is to clean up summary tab and create a relative heat map.

**Comment by weirj1** [comment]
@tleonard6 what is meant by heat map on summary page.  We have a national heat map already.  Don't understand

**Comment by tleonard6** [comment]
@weirj1 For example, let's say you want to see installation impact for 30 day projection.  You select 30 days (from list of 7/14/30/60) and you want to see on a map each installation where both IHME and CHIME peaks > # beds.  heat map is wrong terminology, maybe impact map, bed shortage map, etc. But that's the idea.

**Comment by galarcon0308** [comment]
Once we finish the hospital load calculation for the charts, this should be doable. We might to consider a fixed radius to save computation time if it becomes an issue. With running CHIME for every base or MAJCOM every time the radius is changed

**Comment by tleonard6** [comment]
@galarcon0308 Yep, fixed radius of 50 miles (same as summary table).  Looking for 30 day forecast first, then the others once it's in place.
